src/migrations.js

```diff
@@ -24,7 +21,7 @@ require.scopes.migrations = (function() {
 var exports = {};
 exports.Migrations= {
   changePrivacySettings: function() {
-    if (!chrome.extension.inIncognitoContext) {
+    if (!chrome.extension.inIncognitoContext && chrome.privacy ) {
```

Comment:
So even in private browsing mode in firefox, scripts in the popup are report `inIncognitoContext === false`. Also leaving this here for posterity https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/Extension/inIncognitoContext